helpers/utils.js

```diff
@@ -109,9 +114,8 @@ const playSong = (gameSession, guildPreference, db, message) => {
     };
 
     const songLocation = `${CACHE_DIR}/${gameSession.getLink()}.mp3`;
-    let isSongCached = true;
     if (!fs.existsSync(songLocation)) {
-        isSongCached = false;
+        gameSession.isSongCached = false;
```

Comment:
Why does `isSongCached` need to be part of `GameSession`? Does it need to exist beyond the scope of this function?

Reply:
Yup, in volume.js. It's necessary for changing volume while playing back.